src/main/java/de/uni_tuebingen/ub/nppm/model/PersonHatStand.java

```diff
@@ -0,0 +1,46 @@
+package de.uni_tuebingen.ub.nppm.model;
+
+import javax.persistence.*;
+import java.util.*;
+
+@Entity
+@Table(name = "person_hatstand")
+public class PersonHatStand {
```

Comment:
Should this class also have a _MM suffix at the end? (like e.g. PersonAreal_MM, PersonEthnie_MM, and others)

Reply:
No